internal/storage/round_tripper.go

```diff
@@ -0,0 +1,13 @@
+package storage
```

Comment:
please call the file as user_agent_round_tripper.go

Reply:
fixed